CMake/resolve_dependency_modules/folly/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@
 project(Folly)
 cmake_minimum_required(VERSION 3.28)
 
-velox_set_source(fastfloat)
-velox_resolve_dependency(fastfloat CONFIG REQUIRED)
+velox_set_source(FastFloat)
+velox_resolve_dependency(FastFloat CONFIG REQUIRED)
```

Comment:
without this it's impossible to use system fast float.
Because only FindFastFloat.cmake provided

Reply:
The https://github.com/facebookincubator/velox/blob/main/CMake/resolve_dependency_modules/fastfloat.cmake#L26 name it to FASTFLOAT, which name should we use? I'm not familiar with the Velox build.

Reply:
The fastfloat.cmake will replace FindFastFloat.cmake

Reply:
I think there're two cases, both checked in Velox CI:
1. installed system fast-float, in such case FindFastFloat.cmake will be used
2. from source code build fast-float, in such case fastfloat.cmake will be used, because `velox_build_dependency` is case insensitive. 

> https://github.com/facebookincubator/velox/blob/main/CMake/resolve_dependency_modules/fastfloat.cmake#L26 name it to FASTFLOAT, 

This is expected, because for `velox_resolve_dependency_url` argument should be written in UPPER_CASE